### DISCUSSION TITLE
getStaticProps with Pagination

### DISCUSSION THREAD

**Discussion by joshuaoas** [vercel/next.js#11475] [Help] [answered]
First of all, the static blog starter with tailwind example is great! I've been toying around with it, and I'm not sure how to go about pre-rendering paginated static pages. Do you have any plans to to include something like that in the example, or does anyone have a repo example with this feature?

**Comment by sandys** [comment, marked as answer]
@joshuaoas there is another discussion on same topic - https://github.com/zeit/next.js/discussions/10987

**Reply by joshuaoas** [reply]
Thanks!
